docs/quick-start.md

```diff
@@ -2,7 +2,7 @@
 
 To see how Argo Workflows work, you can install it and run examples of simple workflows.
 
-Before you start you need a Kubernetes cluster and `kubectl` set up to be able to access that cluster. For the purposes of getting up and running, a local cluster is fine. You could consider the following local Kubernetes cluster options:
+Before you start you need a Kubernetes cluster and `kubectl` set up to be able to access that cluster. To get up and run the argo workflow, a local cluster is fine. You could consider the following local Kubernetes cluster options:
```

Comment:
I don't think this reads better.  The prior text is clearer and worded better IMO.

Reply:
Thanks for that. Reverted.

Reply:
Agreed that the prior text was better. Technically speaking, they both actually fail the k8s style guide on [avoiding jargon](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-jargon-and-idioms). I might say "For quick start purposes" to match the existing language / title of the page.

Some of the docs I have yet to touch in general could use a bit of simplification ([per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)).

Reply:
Made some significant simplifications as mentioned above in #12772